### DISCUSSION TITLE
Omega-Package 1.0

### DISCUSSION THREAD

**Discussion by dav-sea** [ltd-profit/omega-package#203] [General]
Давайте подумаем о том что должно быть в Omega-Package
Этот пакет будет содержать самые часто используемые нами утилиты и инструменты для работы с юнити. Необходимо сформировать список функционала который мы должны будем сделать к релизу этого пакета. Жду ваши предложения и пожелания

### Требования к Omega-Package
- В нем должны быть тулзы для юнити и не должно быть тулзов для пакетов, ассетов и модулей
- Он не должен иметь зависимостей 
- Максимальное внимание к обратной совместимости и корректности работы

### Что точно будет сделано в Omega-Package 1.0?  #202
- Omega-Routines будут вынесены в отдельный пакет
- Интерфейс IPool и очевидные реализации

**Comment by zun010** [comment]
- Может быть UIBuilder?
- Мне кажется ещё было бы не плохо иметь некий универсальный ассет для поиска по какому-то контейнеру с фильтрами. Например его можно было бы применить для поиска по дашборду, панели префабов, иерархии, и любых других проектах он был бы совсем не лишним.
- Можно было бы и тултипы доработать и вынести в пакет

**Reply by dav-sea** [reply]
UIBuilder - мне кажется можно вынести в отдельный пакет. Слишком уж жирная штука

> некий универсальный ассет для поиска по какому-то контейнеру с фильтрам

Как это может выглядеть?

> Можно было бы и тултипы доработать и вынести в пакет

Так и нужно сделать, только в отдельный пакет

**Reply by zun010** [reply]
> Как это может выглядеть?

Вообще мне уже не нравится эта идея )0))
